### DISCUSSION TITLE
Simpler cursor shape

### DISCUSSION THREAD

**Discussion by Benni3-1415** [ubuntu/yaru#2867] [Ideas]
Some Ideas (they aren't pixel perfect yet):

### Default cursor
Remove the arrow line at the bottom of the default cursor and increase the size by 1px:
![Simpler concept design for the default cursor](https://user-images.githubusercontent.com/73243667/119844748-fe2df380-bf08-11eb-832c-671f1c81d87c.png "Simpler concept design for the default cursor")
![Simpler concept design for the default cursor (24px)](https://user-images.githubusercontent.com/73243667/119844985-30d7ec00-bf09-11eb-993b-58a841a2e314.png "Simpler concept design for the default cursor (24px)")

#### Benefits
It's an easier to understand the cursor without the irritating details at the bottom.
This design looks more modern and is also used by Adobe:
![The cursor of Adobe Illustrator](https://user-images.githubusercontent.com/73243667/119694616-aaaa9f80-be4d-11eb-8628-fc9b9e971647.png "The cursor of Adobe Illustrator")

### Hand cursor
Remove the fingers and replace them with a curve:
![Quick and dirty concept design](https://user-images.githubusercontent.com/73243667/119694684-bd24d900-be4d-11eb-9e23-7abb88d1bdb6.png "Quick and dirty concept design")
![Quick and dirty concept design (24px)](https://user-images.githubusercontent.com/73243667/119694764-d29a0300-be4d-11eb-8e54-09bea6a7e4cf.png "Quick and dirty concept design (24px)")

#### Benefits
The cursor has no irritating details.
This design looks more modern and fits better to the  rest of the cursor theme.

[go to Pull Request >](https://github.com/ubuntu/yaru/pull/2868#issue-654999991 "Simpler cursor shape")

**Comment by Feichtmeier** [comment]
Neat idea - could you provide a test PR?

**Reply by Benni3-1415** [reply]
Yes, I'm working on it ...

**Reply by Gomesz785** [reply]
> Yes, I'm working on it ...

any update?

**Reply by Benni3-1415** [reply]
> any update?

The [PR](https://github.com/ubuntu/yaru/pull/2868) has already been merged, but the [default cursor has been reverted](https://github.com/ubuntu/yaru/issues/2920).

**Reply by Gomesz785** [reply]
> > any update?
> 
> The [PR](https://github.com/ubuntu/yaru/pull/2868) has already been merged, but the [default cursor has been reverted](https://github.com/ubuntu/yaru/issues/2920).

Thanks. Can you change the part Contra to Cons?
Contra sounds like contraband or some sort of malware/spyware.

**Comment by Benni3-1415** [comment]
## Another (crazy) idea: 4 finger hand
![Quick and dirty _4 finger-design_](https://user-images.githubusercontent.com/73243667/120006588-53393a80-bfd9-11eb-9291-4f5d63285e32.png "Quick and dirty _4 finger-design_")

### Pro
- lower detail level / looks cleaner on small displays
  ![Quick and dirty _4 finger-design_ (24px)](https://user-images.githubusercontent.com/73243667/119998839-60522b80-bfd1-11eb-8718-d76843a47626.png "Quick and dirty _4 finger-design_ (24px)")
- better balanced pointer cursor
  ![Quick and dirty _4 finger-design_ pointer](https://user-images.githubusercontent.com/73243667/120004848-88dd2400-bfd7-11eb-8e85-8fec96fe9aa9.png "Quick and dirty _4 finger-design_ pointer")

- is also used in comics ...

### Cons
- the default line-width of 2px is interrupted (3px)
- unfamiliar hand

_I'm not completely convinced yet_

**Comment by Benni3-1415** [comment]
## My last suggestion
![zoom-in-design](https://user-images.githubusercontent.com/73243667/120005445-1f114a00-bfd8-11eb-9b24-52beeea9ad39.png) ![zoom-out-design](https://user-images.githubusercontent.com/73243667/120005454-20db0d80-bfd8-11eb-8a16-d17e1765f1ce.png)

### Pro
- fits better with the theme
- no other cursor has a large white face
- the default line-width of 2px is interrupted by the dark outline (1px)
- the white `+` or `-` fits with the white triangle in the pencil cursor

---

Should I include the two new changes in the PR too?
